setup.py

```diff
@@ -25,38 +29,64 @@ def package_files(directory):
     "../mlflow/temporary_db_migrations_for_pre_1_users/alembic.ini",
 ]
 
+requirements = [
+    "click>=7.0",
+    "cloudpickle",
+    "databricks-cli>=0.8.7",
+    "requests>=2.17.3",
+    "six>=1.10.0",
+    "python-dateutil",
+    "protobuf>=3.6.0",
+    "gitpython>=2.1.0",
+    "pyyaml",
+    "querystring_parser",
+    "simplejson",
+    "entrypoints",
+    "gorilla",
+    "pandas",
+    "numpy",
+    "sqlparse",
+]
+
+_is_mlflow_skinny = bool(os.environ.get(_MLFLOW_SKINNY_ENV_VAR))
+logging.debug("{} env var is set: {}".format(_MLFLOW_SKINNY_ENV_VAR, _is_mlflow_skinny))
+
+if not _is_mlflow_skinny:
+    requirements.extend(
```

Comment:
This call to `extend()` contains a lot of duplicate elements that are already in `requirements`. We should either dedupe these or, perhaps preferably, maintain separate lists of requirements.

Reply:
I will dedup